packages/node_modules/samples/browser-call-with-screenshare/index.html

```diff
@@ -44,16 +44,46 @@
       <main>
         <h1>Local Screensharing</h1>
         <p>This sample demonstrates sharing your local screen</p>
+
+        <form id="typeCredentials">
```

Comment:
Based on the naming convention of other forms in the page
```suggestion
        <form id="auth-type">
```